src/Uno.UI/UI/Xaml/HitTestability.cs

```diff
@@ -30,4 +30,6 @@ internal enum HitTestability
 		/// </summary>
 		Visible,
 	}
+
+	internal record struct StalePredicate(Predicate<UIElement> Method, string Name);
```

Comment:
As it's internal, it would be better to avoid using a generic delegate type. You can create a delegate type specialized for this case.